src/store.js

```diff
@@ -3,74 +3,101 @@ import axios from 'axios';
 import thunk from 'redux-thunk'
 
 
-    //////////////////////////////////////////////////////////////////////////
-    /////////////////////////REDUX - ACTION TYPES/////////////////////////////
-    //////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////
```

Comment:
I saw this before, and while it's certainly an opinion thing, I don't really know why we do all this commenting. Just the fact that we need this kind of commenting seems to me to mean that these should be in their own files with descriptive names.